scripts/releaseDocs.ts

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import { resolve } from 'path';
+import { runCommand } from './common';
+
+const pathLib = resolve(__dirname, '../packages/core');
+
+async function prepareCoreRelease() {
+  try {
+    const releaseTag = process.argv[2];
+    console.log('Prepare release docs tag:', releaseTag);
+
+    // Check if the current branch is clean (no staged changes)
+    runCommand(
+      'git diff-index --quiet HEAD --',
+      'You have uncommitted changes. Please commit or stash them before running the release script.',
+    );
+
+    // Increment the Core version
+    const versionCmd = releaseTag === 'latest' ? 'patch' : `prerelease --preid ${releaseTag}`;
+    runCommand(`pnpm --filter @grapesjs/docs exec npm version ${versionCmd} --no-git-tag-version --no-commit-hooks`);
```

Comment:
I guess we don't care about tags here, so `pnpm --filter @grapesjs/docs exec npm version patch --no-git-tag-version --no-commit-hooks` should be enough